bug_actiongroup.php

```diff
@@ -206,19 +208,27 @@
 		case 'UP_STATUS':
 			$f_status = gpc_get_int( 'status' );
 			if( access_has_bug_level( access_get_status_threshold( $f_status, $t_bug->project_id ), $t_bug_id ) ) {
-				if( true == bug_check_workflow( $t_status, $f_status ) ) {
+				if( true == bug_check_workflow( $t_status, $f_status ) ||
+					access_compare_level( current_user_get_access_level(), config_get_access( 'status_enum_workflow' ) ) ) {
```

Comment:
Wouldn't that confuse the person who is setting the workflow or not allow them to test what other users would experience?  I would rather not have a different there.